### PR TITLE
Add vendor to stock location

### PR DESCRIPTION
**Pull request by chhlga**

### CODE REVIEW

**Review by damianlegawiec — APPROVED** [review]

**Review by tniezg — APPROVED** [review]

File: app/serializers/spree/v2/storefront/stock_location_serializer_decorator.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module Spree
```
Comment (damianlegawiec) [inline app/serializers/spree/v2/storefront/stock_location_serializer_decorator.rb]:
```suggestion
module SpreeMultiVendor
```

File: app/serializers/spree/v2/storefront/stock_location_serializer_decorator.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  module V2
+    module Storefront
+      module StockLocationSerializerDecorator
+        def self.prepended(base)
+          base.belongs_to :vendor, serializer: Spree::V2::Storefront::VendorSerializer
```
Comment (damianlegawiec) [inline app/serializers/spree/v2/storefront/stock_location_serializer_decorator.rb]:
```suggestion
          base.belongs_to :vendor, serializer: ::Spree::V2::Storefront::VendorSerializer
```

File: app/serializers/spree/v2/storefront/stock_location_serializer_decorator.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+        end
+      end
+    end
+  end
+end
+
+Spree::V2::Storefront::StockLocationSerializer.prepend(Spree::V2::Storefront::StockLocationSerializerDecorator)
```
Comment (damianlegawiec) [inline app/serializers/spree/v2/storefront/stock_location_serializer_decorator.rb]:
```suggestion
Spree::V2::Storefront::StockLocationSerializer.prepend(SpreeMultiVendor::V2::Storefront::StockLocationSerializerDecorator)
```

File: app/serializers/spree/v2/storefront/stock_location_serializer_decorator.rb (hunk 1 above)
Comment (damianlegawiec) [inline app/serializers/spree/v2/storefront/stock_location_serializer_decorator.rb]: @chhlga please move this file into `app/serializerls/spree_multi_vendor/v2/storefront/stock_location_serializer_decorator.rb`

**Comment by chhlga** [comment]
@damianlegawiec could we backport StockLocationSerializer to Spree-4.0?
This commit: https://github.com/spree/spree/commit/74ea4357948e4a5593358b2aff6627e063a996f2#diff-7b50ea50a205de7617ab2b63caf6cbb5287d0fe5668b373c5b3da728b46cbead

**Review by damianlegawiec — APPROVED** [review]

**Pull request merged**